src/main/kotlin/org/sertain/path/PathController.kt

```diff
@@ -0,0 +1,46 @@
+package org.sertain.path
+
+import jaci.pathfinder.Pathfinder
+import jaci.pathfinder.Trajectory
+import jaci.pathfinder.Waypoint
+import jaci.pathfinder.followers.EncoderFollower
+import jaci.pathfinder.modifiers.TankModifier
+
+object PathController {
+    val pathCoordinates = ArrayList<Triple<Double, Double, Double>>()
+
+    var dt = 0.05
+    var max_vel = 1.7
+    var max_accel = 2.0
+    var max_jerk = 60.0
+
+
+    fun addPath(xcoord: Double, ycoord: Double, radians: Double) {
+        pathCoordinates += Triple(xcoord, ycoord, radians)
+    }
+
+    fun configTrajectory(dt: Double, maximumVelocity: Double, maximumAcceleration: Double, maximumJerk: Double) {
+        this.dt = dt
+        this.max_vel = maximumVelocity
+        this.max_accel = maximumAcceleration
+        this.max_jerk = maximumJerk
+    }
+
+    fun generateEncoderFollowers() {
+        val points = ArrayList<Waypoint>()
+        for (coordset: Triple<Double, Double, Double> in pathCoordinates) {
+            points.add(Waypoint(coordset.first, coordset.second, coordset.third))
+        }
+        // Create Trajectory
+        val config = Trajectory.Config(Trajectory.FitMethod.HERMITE_CUBIC, Trajectory.Config.SAMPLES_HIGH, this.dt, this.max_vel, this.max_accel, this.max_jerk)
```

Comment:
`this` isn't necessary here I think? cc: @SUPERCILEX

Reply:
I'll make the `FitMethod` configuration a boolean, since their are only two possible `FitMethod`s available.

Reply:
Keep it the FitMethod enum as it could easily become more than two in the future and he'll we could even add our own. It's always better to be safe with stuff like this and allow it to expand in the future sans-API changes.